lib/OpenQA/Schema/Result/Comments.pm

```diff
@@ -118,7 +118,10 @@ e.g. 'tag:0123:important:GM' returns a list of '0123', 'important' and 'GM'.
 =cut
 sub tag {
     my ($self) = @_;
-    $self->text =~ /\btag:(?<build>([-.@\d\w]+-)?[@\d\w]+):(?<type>[-@\d\w]+)(:(?<description>[@\d\w]+))?\b/;
+    $self->text =~ /\btag:(?<build>[-.@\d\w]+):(?<type>[-@\d\w]+)(:(?<description>[@\d\w]+))?\b/;
+    # note: calling this 'build' is a slight lie, because it may be
```

Comment:
sorry, I don't get it. what I'm saying is that when the tag includes `VERSION-BUILD`, this method doesn't isolate just the `BUILD` component and return that, as the name 'build' sorta implies. It returns the whole `VERSION-BUILD` string.